### ISSUE TITLE
Peer dependencies: include Webpack ^5.0.0

### ISSUE THREAD

**Issue by RCRalph** [negibouze/html-webpack-pug-plugin#25]
In my project I'm using Webpack 5.18.0.

After installing this package, **npm** throws this warning:
```
html-webpack-pug-plugin@2.0.0 requires a peer of webpack@^3.0.0 || ^4.0.0 but none is installed. You must install peer dependencies yourself.
```

From my experience, this package works fine. Is there a possibility of including webpack@^5.0.0 as well?

Thanks for reading and have a nice day :)

**Comment by negibouze** [comment]
Thank you for your report & Pull request.

**Comment by negibouze** [comment]
@RCRalph 

I released a new version(v2.0.1).
Please try it.

**Comment by RCRalph** [comment]
@negibouze 

Works fine for me and doesn't throw the warning.
Thank you and have a nice day :)
